index.html

```diff
@@ -0,0 +1,268 @@
+            </div>
+          </div>
+          <div class="photo"><img src="images/process-4.png" class="photo_image"></div>
+        </div>
+      </div>
+    </div>
+    <!-- ######################################### PROJECT SECTION ######################################### -->
+    <div class="project_container">
+      <div class="project_container_background">
+        <img src="images/circle-2.png" class="circle">
+        <img src="images/bubble-1.png" class="bubble1">
+      </div>
+      <div class="project_container_content">
+        <div class="project_title">
+          <h7>Our Project</h7>
+          <h1>Featured Designs</h1>
+        </div>
+        <div class="project_pages">
+          <div class="project_container_content_left pages">
+            <img src="images/project-1.png" class="project_photos">
+            <div class="project_content_description">
+              <h2>Creative Agency</h2>
+              <h3>Full Branding, Website, App</h3>
+            </div>
+          </div>
+          <div class="project_container_content_right pages">
+            <img src="images/project-2.png" class="project_photos">
+            <div class="project_content_description">
+              <h2>Digital Marketing</h2>
+              <h3>Logo, Website & App</h3>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- ######################################### TOGETHER SECTION ######################################### -->
+    <div class="together_container">
+      <div class="together_background">
+        <img src="images/bubble-12.png" class="together_photo bubble12">
+        <img src="images/bubble-13.png" class="together_photo bubble13">
+        <img src="images/bubble-14.png" class="together_photo bubble14">
+        <img src="images/bubble-16.png" class="together_photo bubble16">
+        <img src="images/chat-group.png" class="together_photo chat">
+        <img src="images/bill-pay.png" class="together_photo bill">
+        <img src="images/laptop-poses.png" class="together_photo laptop">
+      </div>
+      <div class="together_title">
+        <h7>Let's Work Together</h7>
+        <h1>Need a successful project?</h1>
+      </div>
+      <div class="together_project">Estimate Project</div>
+    </div>
+    <!-- ######################################### FOOTER SECTION ######################################### -->
+    <div class="footer_container">
+      <div class="social_bar">
+        <img src="images/facebook.png">
+        <img src="images/twitter.png">
+        <img src="images/pinterest.png">
+        <img src="images/linkedin.png">
+        <img src="images/instagram.png">
+        <img src="images/yt.png">
```

Comment:
Jak się wchodzi na stronę i widzi ikony mediów społecznościowych to zazwyczaj to są linki, które kierują do tych profili. Tutaj też bym widział to na znacznikach `<ul>`, `<a>`.